deployment/docker/docker-compose.yml

```diff
@@ -23,7 +23,7 @@ services:
 
   forms-flow-forms:
     container_name: forms-flow-forms
-    image: formsflow/forms-flow-forms:v7.1.0-alpha
+    image: formsflow/forms-flow-forms:v7.1.0
 
```

Comment:
can you please check & update the forms-flow-forms version in **forms-flow-forms/docker-compose.yml file** too